Build fails without jcenter repository

<!--
NOTE: please submit only bug reports here, any new questions or feature requests should be submitted in Discussions:
https://github.com/software-mansion/react-native-screens/discussions
 -->

## Description
* checkout the project source code
* run yarn
* cd android; ./gradlew assembleRelease

### Expected behavior

```
BUILD SUCCESSFUL
```

### Actual behavior
```
❯ gr assembleRelease
> Task :generateReleaseRFile FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generateReleaseRFile'.
> Could not resolve all files for configuration ':releaseCompileClasspath'.
   > Could not find com.facebook.yoga:proguard-annotations:1.14.1.
     Required by:
         project : > com.facebook.react:react-native:0.63.2
   > Could not find com.facebook.fbjni:fbjni-java-only:0.0.3.
     Required by:
         project : > com.facebook.react:react-native:0.63.2
```

## Package versions
3.6.0, also current master (2de1ac16ddac4ddf3e42d133c130556a73975ba0)


Adding the jcenter repository again, fixes the issue.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,6 +1,7 @@
 buildscript {
     repositories {
         google()
+        jcenter()
         mavenCentral()
     }
     dependencies {
@@ -40,6 +41,7 @@ repositories {
     mavenCentral()
     mavenLocal()
     google()
+    jcenter()
 }
 
 dependencies {

